lib/index.js

```diff
@@ -1,10 +1,12 @@
 /**
  * @file lib/index.js
  */
+const { spawn } = require('child_process');
 const util = require('util');
-const through2 = require('through2');
+
 const colors = require('ansi-colors');
-const { spawn } = require('child_process');
+const through2 = require('through2');
```

Comment:
I did a rearrange here, I'm always influenced by my Python bg, so I prefer to do built-in libs group then 3rd party libs group in my requires (alphabetically).